packages/admin/src/targetGroups/allAssignedContacts/AssignedContactsGrid.gql.ts

```diff
@@ -0,0 +1,24 @@
+import { gql } from "@apollo/client";
+
+const targetGroupBrevoContactsFragment = gql`
+    fragment TargetGroupBrevoContactsList on BrevoContact {
+        id
+        createdAt
+        modifiedAt
+        email
+        emailBlacklisted
+        smsBlacklisted
+    }
+`;
+
+export const assignedBrevoContactsGridQuery = gql`
+    query AssignedBrevoContactsGrid($offset: Int, $limit: Int, $email: String, $targetGroupId: ID!, $brevoId: Float) {
```

Comment:
I would suggest to name this Query `AssignedBrevoContactsGrid` to easier differentiate between the 2 similar queries

Reply:
I changed it to `AllAssignedBrevoContactsGrid` (I guess, you meant that as well :) )

Reply:
Yes that's exactly what i ment! I added the 'All' in my mind I guess 😂